diff --git a/Makefile b/Makefile
index 12e2d10..964cb4c 100644
--- a/Makefile
+++ b/Makefile
@@ -156,7 +156,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 install_sh = /Users/hvoicu/ClionProjects/pydusa07/pydusa-1.15-sparx-8/install-sh
-libdir = ${exec_prefix}/lib
+libdir = ${exec_prefix}
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
 localstatedir = ${prefix}/var
