diff --git a/Makefile.am b/Makefile.am
index 80a4fdb..20949da 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -6,9 +6,9 @@ test:
 
 install:
 	make
-	mkdir -p ${prefix}/lib
-	echo "Adding mpi.so to ${prefix}/lib"
-	cp src/mpi.so ${prefix}/lib
+	mkdir -p ${prefix}
+	echo "Adding mpi.so to ${prefix}"
+	cp src/mpi.so ${prefix}
 	mkdir -p ${prefix}/mpi_examples
 	echo "Installing mpi examples"
 	cp -r mpi_tests/* ${prefix}/mpi_examples/
