diff --git a/Makefile.in b/Makefile.in
index 5a6dacb..8234563 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -156,7 +156,7 @@ htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
-libdir = @libdir@
+libdir = @exec_prefix@
 libexecdir = @libexecdir@
 localedir = @localedir@
 localstatedir = @localstatedir@
@@ -588,9 +588,9 @@ test:
 
 install:
 	make
-	mkdir -p ${prefix}/lib
-	echo "Adding mpi.so to ${prefix}/lib"
-	cp src/mpi.so ${prefix}/lib
+	mkdir -p ${prefix}
+	echo "Adding mpi.so to ${prefix}"
+	cp src/mpi.so ${prefix}
 	mkdir -p ${prefix}/mpi_examples
 	echo "Installing mpi examples"
 	cp -r mpi_tests/* ${prefix}/mpi_examples/
